app/views/random/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= @docs %>
```

Comment:
I updated the code and moved the tests from feature to controller directory to test for json response. Thanks